blues/application/tasks.py

```diff
@@ -224,24 +224,53 @@ def generate_nginx_conf(role='www'):
         f.write(conf)
 
 
-def notify_deploy(role=None):
+def get_github_org():
+    from .project import git_repository
+
+    url = git_repository().get('url', '')
+    if url or not url.startswith('git@github.com:'):
+        return None
+
+    domain, repo = url.split(':')
+    org, repo = repo.split('/')
```

Comment:
If talking about repo path, then it's about `project.git_repository`, not `git.parse_url`.

I believe it's totally fine if a part of dictionary from `git.parse_url` is optional and it would be great to get that info from parse function instead of doing it somewhere else. Could you comment on it, @lundberg ?